NickvisionMoney.GNOME/Views/MainWindow.cs

```diff
@@ -384,6 +373,20 @@ private async void AccountAdded(object? sender, EventArgs e)
         await newAccountView.StartupAsync();
     }
 
+    /// <summary>
+    /// Occurs when the window is shown
+    /// </summary>
+    /// <param name="sender">Gtk.Widget</param>
+    /// <param name="e">EventArgs</param>
+    private async void OnShow(Gtk.Widget sender, EventArgs e)
+    {
+        if (_controller.FileToLaunch != null)
+        {
+            await _controller.AddAccountAsync(_controller.FileToLaunch);
```

Comment:
Just a note: I guess when GLib.AddTimeout will be implemented we'll be able to fix animation by wrapping this line in that function with ~500ms delay added.

Reply:
Just thought of this...can you marshall `g_timeout_add()`?

Reply:
I'm not sure how to properly write a delegate for this...

Reply:
```csharp
private delegate bool GSourceFunc(nint data);

[LibraryImport("libadwaita-1.so.0", StringMarshalling = StringMarshalling.Utf8)]
private static partial uint g_timeout_add(uint interval, [MarshalAs(UnmanagedType.FunctionPtr)]GSourceFunc function, nint data);
```